src/context/walletContext/zeroDevContext.context.tsx

```diff
@@ -79,250 +78,253 @@ const ZeroDevContext = createContext<ZeroDevContextType | undefined>(undefined)
  */
 export const ZeroDevProvider = ({ children }: { children: ReactNode }) => {
 
-    ////// context props
-    //
+  ////// context props
+  //
+
+  ////// ZeroDev props
+  //
+
+  const [isRegistering, setIsRegistering] = useState<boolean>(false)
+  const [isLoggingIn, setIsLoggingIn] = useState<boolean>(false)
+  const [isSendingUserOp, setIsSendingUserOp] = useState<boolean>(false)
+
+  const [kernelClient, setKernelClient] = useState<AppSmartAccountClient | undefined>(undefined)
+  const [isKernelClientReady, setIsKernelClientReady] = useState<boolean>(false)
+  const [address, setAddress] = useState<string | undefined>(undefined)
+
+  const publicClient = createPublicClient({
+    transport: http(consts.BUNDLER_URL),
+    chain: consts.PEANUT_WALLET_CHAIN,
```

Comment:
public client was missing chain as well that was breaking type for kernelClient, only changes in this file